_docs/platforms/docker/install.md

```diff
@@ -7,10 +7,49 @@ redirect_from: /docs/install/docker/container
 redirect_from: install/docker/container
 ---
 
-# Install with Docker 1.10+
+# Installation
 
-If you are running containers without orchestration, you can create a systemd
-service to run the StorageOS container on each node.
+* Ensure that our prerequisites are met, paying particular attention to [system
+  configuration]({% link _docs/prerequisites/systemconfiguration.md %}) and [mount
+  propagation]({% link _docs/prerequisites/mountpropagation.md %})
+* Run a docker container as follows:
+```bash
+/usr/bin/docker run \
+    --name=storageos \
+    --env=DISABLE_TELEMETRY=true \
+    --env=HOSTNAME=$(hostname) \
+    --env=ADVERTISE_IP=127.0.0.1 \
+    --env=JOIN=127.0.0.1 \
+    --net=host \
+    --pid=host \
+    --privileged \
+    --cap-add=SYS_ADMIN \
+    --device=/dev/fuse \
+    --volume=/var/lib/storageos:/var/lib/storageos:rshared \
+    --volume=/run/docker/plugins:/run/docker/plugins \
+    --volume=/sys:/sys
```

Comment:
There is one \ missing at the end of this line

Reply:
Nice catch - thanks!